api/android/build-android-lib.sh

```diff
@@ -341,6 +349,14 @@ if [[ $enable_tflite == "yes" ]]; then
     tar -xJf ./external/tensorflow-lite-$tf_lite_ver.tar.xz -C ./api/src/main/jni
 fi
 
+
+if [[ $enable_decoder_flatbuf == "yes" ]]; then
+    sed -i "s|ENABLE_DECODER_FLATBUF := false|ENABLE_DECODER_FLATBUF := true|" api/src/main/jni/Android.mk
+    sed -i "s|FLATBUF_VER := @FLATBUF_VER@|FLATBUF_VER := ${decoder_flatbuf_ver}|" api/src/main/jni/Android-dec-flatbuf.mk
+    wget --directory-prefix=./external https://raw.githubusercontent.com/nnstreamer/nnstreamer-android-resource/master/external/flatbuffers-${decoder_flatbuf_ver}.tar.xz
```

Comment:
How about giving a `-N` option to prevent re-download of the already existing files?

Reply:
@gichan-jang Neither ```-N``` or ```-nc``` could not handle the cases when the downloading is canceled in the middle of it. Therefore, I just added ```rm``` to remove all ```*.tar.gz``` and ```*.tar.xz``` before the downloading gets started.